typegate/tests/runtimes/wasm/wasm_test.ts

```diff
@@ -0,0 +1,176 @@
+// Copyright Metatype OÜ, licensed under the Elastic License 2.0.
+// SPDX-License-Identifier: Elastic-2.0
+
+import { BasicAuth, tgDeploy } from "@typegraph/sdk/tg_deploy.js";
+import { gql, Meta } from "test-utils/mod.ts";
+import { testDir } from "test-utils/dir.ts";
+import { tg } from "./wasm.ts";
+import * as path from "std/path/mod.ts";
+
+const cwdDir = path.join(testDir, "runtimes/wasm");
+const auth = new BasicAuth("admin", "password");
+
+// Meta.test("Wasm runtime", async (t) => {
```

Comment:
deadcode?

Reply:
@destifo I guess once you find a workaround this can be uncommented.

Reply:
Yeah, I am working on it now.